lib/commands/connect/shared.js

```diff
@@ -182,6 +182,19 @@ exports.withConnection = function(context, heroku) {
   });
 }
 
+var withMapping = exports.withMapping = function(connection, object_name) {
+  return Q.Promise(function(resolve, reject) {
+    var found = false, object_name_lower = object_name.toLowerCase();
```

Comment:
Actually, on further review, the `found` isn't even used anymore. It returns immediately when it resolves now, so there's no more need. I'll still try to watch for that going forward, though.